ci-operator/config/openshift/logging-view-plugin/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
```

Comment:
The reviewer list likely should at a minimum include members of the logging team